recipe/meta.yaml

```diff
@@ -28,15 +26,13 @@ requirements:
     - pillow
     - psutil
     - pyopengl
-    - pyqt >=5.6
-    - {{ pin_compatible('python') }}
```

Comment:
keep this as just ``python`` without the pin_compatible logic

Reply:
I think that's taken care of in the change already, isn't it?